lib/node-loader-core.js

```diff
@@ -165,3 +256,18 @@ function constructImpl(loaders, index, defaultImplementation) {
     return impl(...args, nextImpl);
   };
 }
+
+// Final resolved hook implementations: https://nodejs.org/api/esm.html#esm_hooks
+export const resolve = createHook("resolve");
+export const getFormat = createHook("getFormat");
+export const getSource = createHook("getSource");
+export const transformSource = createHook("transformSource");
+// getGlobalPreloadCode is synchronous, which means we can't import the config file in time :'(
+
+// Make tested methods accessible from `global`, rather than exported directly.
+// Otherwise, the cyclic dependency of importing node-loader-core exports
+// into test files results in the exports being undefined.
+global.nodeLoader = global.nodeLoader || {
```

Comment:
When `global.nodeLoader` is already defined, we should add the `setConfigPromise` and `getConfigPath` properties to it.

The use case is related to order of execution when combining multiple loaders together. See https://github.com/node-loader/node-loader-import-maps/blob/1763d2a82fa5eb91df1ac54b548eec205c8cf3d6/lib/node-import-map-loader.js#L46-L52

Reply:
Ah! Thanks for pointing that out! I'll make the change.